test/scripts/openshift-ci/common.sh

```
@@ -1,4 +1,9 @@
 
+# Source operator subscription configurations
+# Use BASH_SOURCE[-1] to get the path of this common.sh file itself, not the calling script
+COMMON_SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[-1]}")" && pwd)"
+source "${COMMON_SCRIPT_DIR}/operator-subscriptions.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./operator-subscriptions.sh was not specified as input (see shellcheck -x).

(SC1091)